back/doc/dependency_decisions.yml

```diff
@@ -172,3 +172,15 @@
     :why: 
     :versions: []
     :when: 2021-03-28 18:10:30.586364420 Z
+- - :approve
+  - brakeman
+  - :who: 
+    :why: 
+    :versions: []
+    :when: 2022-06-17 14:33:21.323236833 Z
```

Comment:
I checked the license, seems like we can´t do this. See https://github.com/presidentbeef/brakeman/blob/main/LICENSE.md

I'm reaching out to them today to check the conditions. If you want this merged asap, probably safer to cut out brakeman from any other changes.

Reply:
I have taken out brakeman until we know more. We can already merge all other changes.